### ISSUE TITLE
Dead link on the web

### ISSUE THREAD

**Issue by phamduchongan93** [treehouses/treehouses.github.io#96]
I found the link problem in **Git issue tutorial**.

https://treehouses.io/#!./pages/vi/github-issues.md
> NOTE: Remember to always sync your fork before starting to work on a new issue. To sync your fork you can follow the process in the **previous step**.

This block contains a dead link. I known that you refer to this link [https://treehouses.io/#!./pages/vi/gitrepositories.md](https://treehouses.io/#!./pages/vi/gitrepositories.md) which has the **Sync Your Fork** section.

**Comment by dogi** [comment]
good catch
